internal/it/util.go

```diff
@@ -190,6 +190,10 @@ func MustClient(client *hz.Client, err error) *hz.Client {
 	return client
 }
 
+func NewUniqueServiceName(service string) string {
```

Comment:
This function seems to be creating object names, can you rename it to `NewObjectName`?

Reply:
addressed in f2a4add